helm_deploy/laa-court-data-adaptor/templates/_environment.tpl

```diff
@@ -14,6 +14,8 @@ env:
       secretKeyRef:
         name: ec-cluster-output
         key: url
+  - name: RAILS_ENV
+    value: 'production'
```

Comment:
Should this be hardcoded as `production`?

Reply:
The idea was to run the cloud version in `production`, and control via related ENV vars, like `SENTRY_ENV` and so on. Happy to change to a different approach though.

Reply:
Seeing a similar approach in [apply](https://github.com/ministryofjustice/laa-apply-for-legal-aid/blob/9087b689771dcfe37db7ee4891be037a8ce5977a/helm_deploy/apply-for-legal-aid/templates/_envs.tpl#L72-L73) and [get paid CCCD](https://github.com/ministryofjustice/Claim-for-Crown-Court-Defence/blob/13a8140c35ba841d9172241fe0c99d1572d68b34/kubernetes_deploy/jobs/migrate.yaml#L20-L21)

Reply:
The main reason why it looks odd to me, is that we have dev, test, uat, stage and production all in the Cloud Platform. So seeing this hardcoded makes me wonder what it is in those envs?

Reply:
If this is the accepted LAA way of doing things, then that's okay.

Reply:
> The main reason why it looks odd to me, is that we have dev, test, uat, stage and production all in the Cloud Platform. So seeing this hardcoded makes me wonder what it is in those envs?

About the dev, test, uat, stage and prod, the difference is defined via k8s secrets/helm values (eg:  SQS queues and API urls)

Reply:
Yea, typically I'd see the real envs to match those here: https://github.com/ministryofjustice/laa-court-data-adaptor/tree/master/config/environments

This is definitely a different conversation.